style.css

```diff
@@ -1124,193 +1109,18 @@ footer > hr {
   }
 
   .nav-menu.active {
-    left: 0;
+    left: 1px;
   }
 
   .line-hor {
     display: block;
     height: 1px;
     width: 327px;
-    left: 0;
-    border-radius: 0;
+    border-radius: 1px;
     background: #fff;
   }
 
   .icons {
     display: none;
   }
 }
-
-@media only screen and (max-width: 667px)  {
-
-  .modal-main{
-    position: absolute; 
-    height: 989px;
-    left: 0px;
-    top: 0px;
-    background: #979493;
-    mix-blend-mode: multiply;
-  }
-
-  .modal-container{
-    height: 85%;
-    width: 335px;
-    top: 60px;
-    border-radius: 0px;
-    margin-left: 8px;
-}
-
```

Comment:
- Great job implementing the popup modal on your portfolio 👍🏽 , everything looks nice on the desktop view but I think your pop up modal on the mobile view isn't responsive. Can we fix this? 🤗 

Here's what I can see compared to the Figma template
| Your Popup  |Figma Popup |
| :---:   | :---: | 
| <img width="433" alt="Rubbish " src="https://user-images.githubusercontent.com/99246359/204088930-718a8a73-d199-4348-a45b-d41316edd775.png"> |<img width="322" alt="Correct Popup" src="https://user-images.githubusercontent.com/99246359/204088925-c542e674-7b77-471c-93c6-d9c2df5e6c27.png">  |